mlir/test/Conversion/ConvertToSPIRV/scf.mlir

```diff
@@ -33,6 +33,17 @@ func.func @if_yield(%arg0: i1) -> f32 {
 }
 
 // CHECK-LABEL: @while
+// CHECK:       spirv.mlir.loop {
+// CHECK:         spirv.Branch ^[[HEADER:.*]](%{{.*}} : i32)
+// CHECK:       ^[[HEADER]]
+// CHECK:         spirv.Store "Function"
```

Comment:
Why is this store important?

Reply:
I think that when we lower a `scf.while` with `scf.yield`, the header block contains `spirv.Store "Function"`, so I included this to ensure that the `while` is lowered properly.

Reply:
Yes, but I'd expect this to be tested by scf to spirv tests, not convert to spirv

Reply:
I see. I can remove them if they are redundant.